### DISCUSSION TITLE
Requirements for 2 dtu's

### DISCUSSION THREAD

**Discussion by wesseli01** [ohAnd/dtuGateway#43] [Q&A]
Hi, I am using a grid with two dtu's. Can you expand the configuration to use 2 dtu devices? The total value (or/abd dtu 1 + dtu 2 vaule) should be visible on the OLED SH1106. Tx. I am using the ESP32 version on the ESP32 WROOM Dev Kit. Tx.

**Comment by ohAnd** [comment]
Hi @wesseli01 ,

could be the next step... 
prequesite: from my side I have no chance to test this configuration. So you have to test and give feedback.

I opened your question as an new feature issue. Maybe you can extend your idea [on this place ](https://github.com/ohAnd/dtuGateway/issues/44) with more details. e.g. should this a constant connection for active and dynamic power setting or only a display. Means can the connection switching every e.g. 30 sec between the 1st and 2nd dtu ... 

And
- which additions for which values should be done and shown
- which powersetting should be shown, only the highest, an average between both, or ...
- ...

Then let's see how we can go further...

BR
Andreas

**Comment by wesseli01** [comment]
> Am 26.08.2024 um 15:14 schrieb Andreas ***@***.***>:
> 
> 
> Hi @wesseli01 <https://github.com/wesseli01> ,
> 
> could be the next step...
> prequesite: from my side I have no chance to test this configuration. So you have to test and give feedback.
> 
I 

> I opened your question as an new feature issue. Maybe you can extend your idea on this place  <https://github.com/ohAnd/dtuGateway/issues/44>with more details. e.g. should this a constant connection for active and dynamic power setting or only a display
> 
only display
> . Means can the connection switching every e.g. 30 sec between the 1st and 2nd dtu …
> 
yes
> And
> 
> which additions for which values should be done and shown
		the same current dtu1 values in the web front end for the 2nd
> which powersetting should be shown, only the highest, an average between both, or ..
		current sum value (dtu1 + dtu2) on display

**Comment by wesseli01** [comment]
> Am 26.08.2024 um 15:14 schrieb Andreas ***@***.***>:
> 
> 
> Hi @wesseli01 <https://github.com/wesseli01> ,
> 
> could be the next step...
> prequesite: from my side I have no chance to test this configuration. So you have to test and give feedback.
> 
I can test it.

> I opened your question as an new feature issue. Maybe you can extend your idea on this place  <https://github.com/ohAnd/dtuGateway/issues/44>with more details. e.g. should this a constant connection for active and dynamic power setting or only a display
> 
only display
> . Means can the connection switching every e.g. 30 sec between the 1st and 2nd dtu …
> 
yes
> And
> 
> which additions for which values should be done and shown
		the same current dtu1 values in the web front end for the 2nd
> which powersetting should be shown, only the highest, an average between both, or ..
		current sum value (dtu1 + dtu2) on display
